xbmc/cores/VideoPlayer/DVDDemuxSPU.cpp

```diff
@@ -365,7 +365,7 @@ CDVDOverlaySpu* CDVDDemuxSPU::ParseRLE(CDVDOverlaySpu* pSPU, uint8_t* pUnparsedD
   unsigned int i_x, i_y;
 
   // allocate a buffer for the result
-  unsigned __int16* p_dest = (unsigned __int16*)pSPU->result;
+  uint16_t* p_dest = (uint16_t*)pSPU->result;
```

Comment:
On 2016/03/24 21:34, Garrett Brown <notifications@github.com> wrote:
> > @@ -365,7 +365,7 @@ CDVDOverlaySpu* CDVDDemuxSPU::ParseRLE(CDVDOverlaySpu* pSPU, uint8_t* pUnparsedD
> >    unsigned int i_x, i_y;
> >  
> >    // allocate a buffer for the result
> > -  unsigned __int16* p_dest = (unsigned __int16*)pSPU->result;
> > +  uint16_t* p_dest = (uint16_t*)pSPU->result;
> 
> mind changing to reinterpret_cast while you're at it?

Yes, but IMHO such things have to be done in a different commit.  It
would be confusing to do two things at once, it would be more
difficult to review.